cookbook/core/flyte_basics/task_cache.py

```diff
@@ -13,6 +13,9 @@
 
 
 # %%
+# Task Cache Default
```

Comment:
Rephrase - this makes it sound like we add a default value to the cache.

Reply:
@cosmicBboy how would you like to rephrase?